components/com_joomgallery/models/forms/ajaxupload.xml

```diff
@@ -109,6 +109,12 @@
         description="COM_JOOMGALLERY_DEBUG_MODE_ASTERISK"
         value="1"
       />
+      <field
+        name="ajaxupload"
+        type="fineuploader"
+        form_id="AjaxUploadForm"
+        field_id_prefix="ajax_"
+      />
```

Comment:
I think there is missing the ```redirect``` attribute because the redirect after upload in the frontend is not working yet.